src/apps/loan-list/materials/utils/status-badge-content.tsx

```diff
@@ -0,0 +1,29 @@
+import clsx from "clsx";
+import React, { FC } from "react";
+
+interface StatusBadgeContentProps {
+  statusLabelType: "danger" | "warning" | "neutral" | "info";
+  text: string;
+  mobileVersion?: boolean;
```

Comment:
Great with a separate component!
I think we decided on having a naming convention around booleans?
So in that case it should be: `isMobileVersion `?